lib/calculator.html

```diff
@@ -0,0 +1,32 @@
+<div class="row">
```

Comment:
The HTML for the calculator component. Notice how there is no remove button as that is not technically part of the calculator. That is the responsibility of the App / Dashboard that hosts the calculators.